lib/remote_persistent_term/fetcher/s3.ex

```diff
@@ -113,7 +122,11 @@ defmodule RemotePersistentTerm.Fetcher.S3 do
     end
   end
 
-  defp find_latest(_, _), do: {:error, :not_found}
+  defp find_latest(_), do: {:error, :not_found}
```

Comment:
Are you matching on this :not_found somewhere? Otherwise I think we should either:
- use a more descriptive string, something like "object @ #{key} was not found"
- introduce an Exception that will produce a similar message to the string above

Reply:
yes we match on it already when we call find_latest:

```
      {:error, :not_found} ->
        {:error, "could not find s3://#{state.bucket}/#{state.key}"}
```

Reply:
perfect